src/viewers/SideBySideViewer.js

```diff
@@ -73,7 +80,10 @@ const SideBySideViewer = props => {
     sliderValues,
     colorValues,
     channelIsOn,
-    loaderSelection,
+    loaderSelection: oldLoaderSelection,
+    newLoaderSelection,
+    onViewportLoad,
+    transitionFields: GLOBAL_SLIDER_DIMENSION_FIELDS,
```

Comment:
I think we could make this a react prop and have it default to `GLOBAL_SLIDER_DIMENSION_FIELDS` so that others could change it in the future, for example if they had a slider for going between images registered along some other axis (imaging angles, maybe?)